### ISSUE TITLE
Clustered Mgt server: on upgrade another mgt server is detected with the same ip

### ISSUE THREAD

**Issue by remibergsma** [MissionCriticalCloud/cosmic#472]
After which it shuts itself down. A simple `systemctl restart tomcat` solves the issue.

![image](https://user-images.githubusercontent.com/1630096/27341615-1441bf6e-55de-11e7-8c97-f86ac2fe69d1.png)

```
2017-06-20 17:27:26.544 ERROR [c.c.c.ClusterManagerImpl] Detected that another management node with the same IP a.x.y.z is already running, please check your cluster configuration
2017-06-20 17:27:26.544 ERROR [c.c.s.l.CloudStackExtendedLifeCycle] Failed to configure ClusterManagerImpl
javax.naming.ConfigurationException: Detected that another management node with the same IP a.x.y.z is already running, please check your cluster configuration
        at com.cloud.cluster.ClusterManagerImpl.checkConflicts(ClusterManagerImpl.java:350)
        at com.cloud.cluster.ClusterManagerImpl.configure(ClusterManagerImpl.java:301)
        at com.cloud.spring.lifecycle.CloudStackExtendedLifeCycle$1.with(CloudStackExtendedLifeCycle.java:70)
        at com.cloud.spring.lifecycle.CloudStackExtendedLifeCycle.with(CloudStackExtendedLifeCycle.java:84)
        at com.cloud.spring.lifecycle.CloudStackExtendedLifeCycle.configure(CloudStackExtendedLifeCycle.java:66)
        at com.cloud.spring.lifecycle.CloudStackExtendedLifeCycle.start(CloudStackExtendedLifeCycle.java:37)
        at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:175)
        at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:50)
        at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:348)
        at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:151)
        at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:114)
        at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:879)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:545)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.loadContext(DefaultModuleDefinitionSet.java:123)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.lambda$loadContexts$1(DefaultModuleDefinitionSet.java:101)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.withModule(DefaultModuleDefinitionSet.java:220)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.withModule(DefaultModuleDefinitionSet.java:225)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.withModule(DefaultModuleDefinitionSet.java:225)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.withModule(DefaultModuleDefinitionSet.java:207)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.loadContexts(DefaultModuleDefinitionSet.java:96)
        at com.cloud.spring.module.model.impl.DefaultModuleDefinitionSet.load(DefaultModuleDefinitionSet.java:60)
        at com.cloud.spring.module.factory.ModuleBasedContextFactory.loadModules(ModuleBasedContextFactory.java:19)
        at com.cloud.spring.module.factory.CloudStackSpringContext.init(CloudStackSpringContext.java:52)
        at com.cloud.spring.module.factory.CloudStackSpringContext.<init>(CloudStackSpringContext.java:42)
        at com.cloud.spring.module.factory.CloudStackSpringContext.<init>(CloudStackSpringContext.java:33)
        at com.cloud.spring.module.web.CloudStackContextLoaderListener.contextInitialized(CloudStackContextLoaderListener.java:32)
        at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5068)
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5584)
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:147)
        at org.apache.catalina.core.StandardContext.reload(StandardContext.java:4088)
        at org.apache.catalina.loader.WebappLoader.backgroundProcess(WebappLoader.java:425)
        at org.apache.catalina.core.ContainerBase.backgroundProcess(ContainerBase.java:1342)
        at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1543)
        at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1553)
        at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1553)
        at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.run(ContainerBase.java:1521)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by remibergsma** [comment]
One option seems to be to reduce `cluster.heartbeat.threshold` but that feels like working around it.

**Comment by remibergsma** [comment]
Fixed in Cosmic 6.4.x 🎉
